parser/fsimporter.go

```diff
@@ -77,35 +77,17 @@ func (fs *FSImporter) GetInternalSubnets() []*net.IPNet {
 	return fs.internal
 }
 
-//Run starts the importing
-func (fs *FSImporter) Run() {
-	// track the time spent parsing
-	start := time.Now()
-	fs.res.Log.WithFields(
-		log.Fields{
-			"start_time": start.Format(util.TimeFormat),
-		},
-	).Info("Starting filesystem import. Collecting file details.")
```

Comment:
I ended up removing these log statements. I had them in but they cluttered up the calling function and I started thinking that these particular ones aren't all that valuable currently anyway. We haven't been concerned about the performance of this code in quite some time.